clang/lib/Headers/hlsl/hlsl_intrinsics.h

```diff
@@ -303,6 +303,48 @@ fmod(__detail::HLSL_FIXED_VECTOR<float, N> X,
   return __detail::fmod_vec_impl(X, Y);
 }
 
+//===----------------------------------------------------------------------===//
+// ldexp builtins
+//===----------------------------------------------------------------------===//
+
+/// \fn T ldexp(T X, T Exp)
+/// \brief Returns the result of multiplying the specified value by two, raised
```

Comment:
I think the comma after two changes the meaning of this. To me this reads as (X * 2) ^ exp. 
I would remove the comma.